voxblox/include/voxblox/utils/layer_utils.h

```diff
@@ -13,7 +13,8 @@ namespace voxblox {
 namespace utils {
 
 template <typename VoxelType>
-bool isSameVoxel(const VoxelType& voxel_A, const VoxelType& voxel_B) {
+bool isSameVoxel(const VoxelType& /* voxel_A */,
```

Comment:
why remove the variable names?

Reply:
This keeps giving warnings in clang, since we have warnings for unused variables on.